### PR TITLE
Allow slower move of text using arrow keys by holding Alt

### PR DESCRIPTION
**Pull request by gary-lgy**

### CODE REVIEW

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -388,7 +388,9 @@ Moves the text at index 3 to the right by 20% of the image width and up by 30% o
 
 [TIP]
 You can type `move INDEX` and use arrow keys to move the specified text. +
-Hold kbd:[Shift] while pressing arrow keys to make the text move faster.
+Hold kbd:[Shift] while pressing arrow keys to make the text move faster. +
+Hold kbd:[Alt] while pressing arrow keys to make the text move slower.
```
Comment (moziliar) [inline docs/UserGuide.adoc]: Might want to specify what Alt is on MacOS (which is Option key).

Comment (gary-lgy) [reply docs/UserGuide.adoc]: Have you tried if Option key works?

Comment (moziliar) [reply docs/UserGuide.adoc]: Yes, the increment is reduced. Option behaves like Alt on MacOS.

**Comment by moziliar** [comment]
Might want to add the prompt in command prompter to suggest to the user "you can do this". Can be in another PR tgt with the command validation (It still shows red but with correct command prompt).

**Pull request merged**